### DISCUSSION TITLE
Update SR div2k-dataset documents

### DISCUSSION THREAD

**Discussion by EdwardZhao1991** [open-mmlab/mmediting#1206] [Datasets] [answered]
Hi all,

Thanks for the awesome project.

I'm a beginner at super-resolution and I wand to train an `ESDR` model based on DIV2K dataset.

However, based on the document, I cannot find the `val_set5` on the official [dataset website](https://data.vision.ee.ethz.ch/cvl/DIV2K/).
https://mmediting.readthedocs.io/en/latest/sr_datasets.html#div2k-dataset

1. Could you please tell me how to obtain these files?

2. Moreover, it would be great if you can provide a demo code to generate the annotations.
https://github.com/open-mmlab/mmediting/blob/2ab95c484a1eed48fccdd8a899aa89fbfb6b312f/configs/restorers/edsr/edsr_x2c64b16_g1_300k_div2k.py#L86

3. Where is the code to compute the mean and std?
https://github.com/open-mmlab/mmediting/blob/2ab95c484a1eed48fccdd8a899aa89fbfb6b312f/configs/restorers/edsr/edsr_x2c64b16_g1_300k_div2k.py#L15-L16

> does it consider the zero regions? e.g., in medical images (CT/MR), many pixels are zero.

Thanks again for the great project.

**Comment by Yshuo-Li** [comment, marked as answer]
1. `val_set5` is another dataset called [`Set5`](https://paperswithcode.com/dataset/set5)
2. You can crop images by `https://github.com/open-mmlab/mmediting/blob/master/tools/data/super-resolution/div2k/preprocess_div2k_dataset.py`
3. Mean and std are provided in EDSR. Besides, we cannot give firm advice on whether consider the zero regions in medical images.
